configuration/packages.yaml

```diff
@@ -9,6 +9,11 @@ packages:
     paths:
       openssl@system: /usr/lib64
 
+  jdk:
+    buildable: False
+    paths:
+      jdk@system: /usr
```

Comment:
I think `jdk@system` won't work as the jdk package parses the version string, I had to resort to using:
`jdk@8u141-b16: /usr/lib/jvm/jre`
This matches the current version installed in the system.